tests/ert/ui_tests/cli/conftest.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def reduce_omp_num_threads_count():
+    multithreading_env_variables = [
+        ["OMP_NUM_THREADS", "1"],  # OpenMP
```

Comment:
I think only OMP_NUM_THREADS is necessary to set in order to choke numpy.

Reply:
Okay, we can try only using that 👌

Reply:
^will update commit msg